ipatests/test_cmdline/test_help.py

```diff
@@ -120,6 +122,16 @@ def test_command_help():
     assert unicode(user_add.doc) in help_ctx.stdout
 
 
+def test_command_help_without_cache():
+    cache_dir = os.path.expanduser('~/.cache/ipa/schema/')
+    if os.path.exists(cache_dir):
```

Comment:
You should rather use ``os.path.isdir`` instead of ``os.path.exists``. Or even simpler, ``shutil.rmtree(cache_dir, ignore_errors=True)``